client/my-sites/customer-home/cards/features/blogging-prompt/prompts-navigation.jsx

```diff
@@ -0,0 +1,190 @@
+import { Button, Gridicon } from '@automattic/components';
+import { addQueryArgs } from '@wordpress/url';
+import classnames from 'classnames';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import getEditorUrl from 'calypso/state/selectors/get-editor-url';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import './style.scss';
+
+export class PromptsNavigation extends Component {
+	static propTypes = {
+		prompts: PropTypes.array,
+		editorUrl: PropTypes.string,
+		backIcon: PropTypes.string,
+		forwardIcon: PropTypes.string,
+	};
+
+	static defaultProps = {
+		prompts: null,
+		editorUrl: '',
+		backIcon: 'arrow-left',
+		forwardIcon: 'arrow-right',
+	};
+
+	state = {
+		promptIndex: 0,
+	};
+
+	navigatePrompts = ( direction ) => {
+		if ( direction === 'forward' ) {
+			this.goToNextStep();
+		} else {
+			this.goToPreviousStep();
+		}
+	};
+
+	trackSkippedBloggingPrompt() {
+		const todayPromptId = this.props.prompts[ 0 ].id;
+		const skippedPromptId = this.getPrompt().id;
+		if ( todayPromptId !== skippedPromptId ) {
+			recordTracksEvent( `calypso_customer_home_skip_prompt`, {
+				site_id: this.props.siteId,
+				prompt_id: skippedPromptId,
+			} );
+		}
+	}
+
+	trackBloggingPromptClick() {
+		recordTracksEvent( `calypso_customer_home_answer_prompt`, {
+			site_id: this.props.siteId,
+			prompt_id: this.getPrompt().id,
+		} );
+
+		this.trackSkippedBloggingPrompt();
```

Comment:
I think it would be clearer if the contents of `trackSkippedBloggingPrompt()` where just inlined here. 

At first I thought that it would always record a skipped prompt when clicking the prompt link which is confusing.